### DISCUSSION TITLE
Complete beginner - setting up new shop reports multiple errors

### DISCUSSION THREAD

**Discussion by cob-jj106** [jef/streetmerchant#2934] [Help] [answered]
ELI5 because this is the first github repository i've ever worked with - I only have very minor past coding knowledge from being a UX designer (so nothing).

I apologise if i'm posting this in the wrong place.

This software is exactly what i've been looking for. I'm trying to setup a shop to track the Dyson Hairdryer on [this page](https://www.dyson.co.uk/hair-care/dyson-supersonic/dyson-supersonic-blue-copper) (Present buying is a bitch). I've followed the 'getting started' guide as best as I can with no luck. Can someone dumb it down even further to assist me in simply getting a notifcation when that hairdryer goes in stock please?

Really appreciate your time and knowledge.

Cheers,
J

**Comment by Mr-Tech-13** [comment]
Join the streetmerchant discord and read through #help. This will give you a more dumbed down response and I can answer you faster. https://discord.gg/G6ZqtEZtg2

**Comment by Mr-Tech-13** [comment]
Specifically this

https://discord.com/channels/773913070665859073/776482351102427196/933750485222060132

**Comment by jef** [comment, marked as answer]
this should get you to where you need: https://www.jef.buzz/streetmerchant/help/general/#adding-a-store
